src/apis/rooms/queries.ts

```
@@ -1,15 +1,36 @@
+ * @returns 숙소의 상세 정보
+ */
 export async function getRoom(id: number): Promise<Room> {
-  const response = await httpClient.get<Room>(`/rooms/${id}`);
+  try {
+    const response = await httpClient.get<Room>(`/rooms/${id}`);
+    if (!response.success) {
+      switch (response.status) {
+        case 404:
+          notFound();
+        default:
+          throw new CustomError(response.message, response.status);
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)